package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "forta-agent-tools",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
Instead of `3.2.0`, we can make it `3.1.1`.

Reply:
done 👍